docs/user/gtfs-plus-editor.md

```diff
@@ -90,6 +114,8 @@ Feed transformations are set up in Data Manager outside the GTFS+ Editor, under
 
 #### Using `route_attributes.txt` from another feed version
 
+**Important!** To inject GTFS+ edits done to another version, **the edits need to be published first to a new GTFS version** (see [Publishing a new feed version](#publishing-a-new-feed-version) above). In the last step below, select that published version as the source for the route attributes to reuse.
```

Comment:
I'm a bit confused by this, but maybe if I used the gtfs+ module regularly it would make more sense. Will leave it up to the second reviewer!